src/tabs/home/Home.js

```diff
@@ -72,6 +86,15 @@ class Home extends PureComponent {
           >
             {({ loading, error, data, refetch, fetchMore, variables }) => (
               <FeedView
+                ListItemComponent={({ ...item }) => (
+                  <ContentCardConnected
```

Comment:
I mentioned this in another comment, but by making everything a `ContentCardConnected` in this case, we lose the 2 different card types we currently show in the feed. 

![Screenshot 2020-04-15 14 31 40](https://user-images.githubusercontent.com/3229463/79377165-44084200-7f29-11ea-9c13-89443373f9f2.png)